packages/apps/auth0-database-scripts/src/create.ts

```diff
@@ -34,6 +34,11 @@ async function create(user: Auth0UserWithPassword) {
     user.email,
     user.password
   );
+
+  console.log(
+    `Received create patron response = ${JSON.stringify(createPatronResponse)}`
+  );
```

Comment:
This is because we replace the original response when we throw our `'Please use a more complex password'` error, and it'd be useful to log the underlying Sierra response. It appears in the Auth0 log console.